pkgs/by-name/cl/clickhouse/update.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p jq nix
+
+set -euo pipefail
+echoerr() { echo "$@" 1>&2; }
+
+fetchgithub() {
+    set +eo pipefail
+    nix-build -A "$1" 2>&1 >/dev/null | grep "got:" | cut -d':' -f2 | sed 's| ||g'
+    set -eo pipefail
+}
+
+fname="$1"
+echoerr "Working on $fname"
+shift
+
+# Fetch latest tags from the repo, leave only stable and lts, use version sort in reverse order.
+all_tags=$(curl -s https://api.github.com/repos/ClickHouse/ClickHouse/tags \
```

Comment:
This should expand `GITHUB_TOKEN` where available (see my latest `prometheus` PR) - presumably the NixOS infra has one set with all the API calls it's doing.